command/login.go

```diff
@@ -71,6 +77,7 @@ func (l *LoginCommand) AutocompleteFlags() complete.Flags {
 		complete.Flags{
 			"-method":             complete.PredictAnything,
 			"-oidc-callback-addr": complete.PredictAnything,
+			"-login-token":        complete.PredictNothing,
```

Comment:
```suggestion
			"-login-token":        complete.PredictAnything,
```

It doesn't _really_ matter in this case, but `PredictAnything` calls the predictor function, while `PredictNothing` doesn't.